packages/xs-extra/wsproxy.master/opam

```diff
@@ -7,23 +7,8 @@ license: "LGPL-2.0 with OCaml linking exception"
 homepage: "https://github.com/xapi-project/wsproxy"
 dev-repo: "https://github.com/xapi-project/wsproxy.git"
 bug-reports: "https://github.com/xapi-project/wsproxy/issues"
```

Comment:
Same comment about the `tags: [ "org:xapi-project" ]` tag, to make the opam files match.